opm/autodiff/RedistributeDataHandles.hpp

```diff
@@ -327,7 +325,8 @@ void distributeGridAndData( Dune::CpGrid& grid,
     grid.loadBalance(eclipseState);
     grid.switchToDistributedView();
 
-    distributed_props = std::make_shared<BlackoilPropsAdFromDeck>(properties, grid.numCells());
+    BlackoilPropsAdFromDeck distributed_props = BlackoilPropsAdFromDeck(properties,
+                                                                        grid.numCells());
     distributed_state.init(grid.numCells(), grid.numFaces(), state.numPhases());
```

Comment:
I agree (and prefer the first one), but it is a minor detail. Will you fix it before I merge?